modules/nf-core/samtools/collatefastq/main.nf

```diff
@@ -9,11 +9,13 @@ process SAMTOOLS_COLLATEFASTQ {
 
     input:
     tuple val(meta), path(input)
+    tuple val(meta2), path(fasta)
+    val(interleave)
 
     output:
     //TODO might be good to have ordered output of the fastq files, so we can
     // make sure the we get the right files
-    tuple val(meta), path("*_{1,2}.fq.gz"), path("*_other.fq.gz"), path("*_singleton.fq.gz"), emit: reads
+    tuple val(meta), path("*_{1,2,interleaved}.fq.gz"), path("*_other.fq.gz"), path("*_singleton.fq.gz"), emit: reads
     path "versions.yml"                                                                     , emit: versions
```

Comment:
alignment at comma 🙈

Reply:
surprised this is not complaining about interleaved missing, if false.

Reply:
Why should it? The glob lists possible options but doesn't require them all I think.

Reply:
fixed the alignment 😉